scene/resources/curve.cpp

```diff
@@ -490,8 +504,93 @@ void Curve::ensure_default_setup(real_t p_min, real_t p_max) {
 	}
 }
 
+bool Curve::_set(const StringName &p_name, const Variant &p_value) {
+	Vector<String> components = String(p_name).split("/", true, 2);
+	if (components.size() >= 2 && components[0].begins_with("point_") && components[0].trim_prefix("point_").is_valid_int()) {
+		int point_index = components[0].trim_prefix("point_").to_int();
+		String property = components[1];
+		if (property == "position") {
+			Vector2 position = p_value.operator Vector2();
+			set_point_offset(point_index, position.x);
+			set_point_value(point_index, position.y);
+			return true;
+		} else if (property == "left_tangent") {
+			set_point_left_tangent(point_index, p_value);
+			return true;
+		} else if (property == "left_mode") {
+			// TODO For some reason the compiler won't convert from Variant to enum
+			int mode = p_value;
+			set_point_left_mode(point_index, (TangentMode)mode);
```

Comment:
I think casting to enum is expected, enums are just ints in Variant.

Reply:
The comment kind of already existed in this line:

https://github.com/godotengine/godot/blob/7ea8cde9834b0fda4a928217b7880da3dd330214/scene/resources/curve.cpp#L412

So I thought maybe the intention was to make the casting implicit at some point, but I think it's fine casting explicitly and removing the comment, I'll remove the comment from the other place too then.